### ISSUE TITLE
lizard scanning `gmock` instead of source files

### ISSUE THREAD

**Issue by jhdcs** [sscpac/statick#301]
Due to how the tool plugin collects and passes package information, the lizard tool will always scan the `gmock` source directory, rather than the package source directory (since that's the first 'src_dir' key that the tool finds). See: https://github.com/sscpac/statick/blob/master/statick_tool/plugins/tool/lizard_tool_plugin.py.

Code at issue:
```    def scan(self, package: Package, level: str) -> Optional[List[Issue]]:
        """Run tool and gather output."""
        src_dir = "."
        if "src_dir" in package:
            src_dir = package["src_dir"]```

**Comment by tdenewiler** [comment]
I was able to re-create the issue and I'm pretty sure I have a fix for this. I need to test it more.

**Comment by tdenewiler** [comment]
@jhdcs, I posted my fix in #302. If you have time to review/test those changes I would appreciate it.

**Comment by jhdcs** [comment]
I've reviewed the PR, though since I don't have write access, GitHub thinks it "doesn't count".

**Comment by denewiler** [comment]
Fixed in #302.
